spec/requests/distributions_requests_spec.rb

```diff
@@ -7,7 +7,7 @@
 
   let(:secret_key) { "HI MOM THIS IS ME AND I'M CODING" }
   let(:crypt) { ActiveSupport::MessageEncryptor.new(secret_key) }
-  let(:hashed_id) { crypt.encrypt_and_sign(@organization.id) }
+  let(:hashed_id) { CGI.escape(crypt.encrypt_and_sign(@organization.id)) }
```

Comment:
@dorner could you give a brief explanation on why this fixes it and what caused the original issue?

Reply:
Yep. Looks like the controller does `CGI.unescape` when reading the value, as it's coming through a browser URL. Looks like request specs do *not* automatically call `CGI.escape` when passing parameters as you'd expect, so we have to do that manually.

Reply:
Ahh okay!